e2e/test/provisioning/ProvisioningCertificateValidationE2ETest.cs

```diff
@@ -16,7 +17,7 @@
 namespace Microsoft.Azure.Devices.E2ETests.Provisioning
 {
     [TestClass]
-    [Ignore("TODO: Enable when invalid cert server is back online.")]
+    //[Ignore("TODO: Enable when invalid cert server is back online.")]
```

Comment:
Why still ignored?

Reply:
Good catch, my bad missing this.
Updated with new commit